demo/meeting/src/containers/MeetingDetails/index.tsx

```diff
@@ -28,7 +30,7 @@ const MeetingDetails = () => {
           </div>
           <div>
             <dt>Hosted region</dt>
-            <dd>{region}</dd>
+            <dd>{manager.meetingRegion}</dd>
```

Comment:
Shouldn't this be in a `description list` (eg; `<dl>`)?

Reply:
Oops, missed that one - thanks

Reply:
Actually nvm, the `StyledList` is what renders the dl. The divs are just for grouping/forcing a new line